workers/Dockerfile

```diff
@@ -32,6 +32,13 @@ RUN mkdir -p /var/local/generic-worker
 # autogenerated ed25519 key is only good for local and testing
 RUN generic-worker new-ed25519-keypair --file /etc/generic-worker/ed25519_key
 
+# Write out the DockerFlow-compatible version.json file
+ARG DOCKER_FLOW_VERSION
+RUN if [ -n "${DOCKER_FLOW_VERSION}" ]; then \
+    echo "${DOCKER_FLOW_VERSION}" > /version.json; \
+else \
+    echo \{\"version\": \"44.18.0\", \"commit\": \"local\", \"source\": \"https://github.com/taskcluster/taskcluster\", \"build\": \"NONE\"\} > /version.json; \
+fi
```

Comment:
Should this file be `workers/generic-worker/Dockerfile` rather than `workers/Dockerfile`?

Reply:
This one also includes extra tools, not only generic worker - proxy, livelog, start-worker, taskcluster-cli.. maybe it should be on the top level somewhere?